test/test-platform-output.c

```diff
@@ -102,6 +105,27 @@ TEST_IMPL(platform_output) {
     printf("  times.nice: %llu\n",
            (unsigned long long) cpus[i].cpu_times.nice);
   }
+
+  uv_sleep(100);
+  err = uv_cpu_info(&cpus2, &count);
+  ASSERT(err == 0);
+
+  cpu1_total = cpus[0].cpu_times.user
+               + cpus[0].cpu_times.nice
+               + cpus[0].cpu_times.sys
+               + cpus[0].cpu_times.idle
+               + cpus[0].cpu_times.irq;
+  cpu2_total = cpus2[0].cpu_times.user
+               + cpus2[0].cpu_times.nice
+               + cpus2[0].cpu_times.sys
+               + cpus2[0].cpu_times.idle
+               + cpus2[0].cpu_times.irq;
+
+  // Check that total CPU times has increased by 100ms +/- 25ms
```

Comment:
```suggestion
  /* Check that total CPU times has increased by 100ms +/- 25ms. */
```
That said, I'm not 100% sure if the check is reliable. For example, Linux's CPU governor can, at least in theory, take a CPU offline when it's idle. I suppose we'll just have to see how the test fares on the CI.

Reply:
I hope it's reliable, but happy to pull the test out if it turns out not to be.